controller/controller_server/csi_controller_server.py

```diff
@@ -75,17 +76,41 @@ def GetCapacity(self, request, context):
     def ControllerGetCapabilities(self, request, context):
         logger.debug("ControllerGetCapabilities")
         types = csi_pb2.ControllerServiceCapability.RPC.Type
-                    
+        
+        context.set_code(grpc.StatusCode.OK)
         return csi_pb2.ControllerGetCapabilitiesResponse(
            capabilities=[csi_pb2.ControllerServiceCapability(
                             rpc=csi_pb2.ControllerServiceCapability.RPC(type=types.Value("CREATE_DELETE_VOLUME"))),
                          csi_pb2.ControllerServiceCapability(
                              rpc=csi_pb2.ControllerServiceCapability.RPC(type=types.Value("PUBLISH_UNPUBLISH_VOLUME")))  ])
  
+    def GetPluginInfo(self, request, context):
+        logger.debug("GetPluginInfo")
+        context.set_code(grpc.StatusCode.OK)
+        return csi_pb2.GetPluginInfoResponse(name="ibm-block-csi-driver", vendor_version="1.0.0")
+    
+    def GetPluginCapabilities(self, request, context):
+        logger.debug("GetPluginCapabilities")
+        context.set_code(grpc.StatusCode.OK)
+        types = csi_pb2.PluginCapability.Service.Type
+        return csi_pb2.GetPluginCapabilitiesResponse(
+            capabilities=[csi_pb2.PluginCapability(
+                            service=csi_pb2.PluginCapability.Service(type=types.Value("CONTROLLER_SERVICE")))
```

Comment:
since identity should be done also in python for node, then I think its better also here to put the CONTROLLER_SERVER as one of the consts inside the plugin.conf (maybe like a list IdentityCapabilities=CONTROLLER_SERVER), and just use it from here.

Reply:
why should identity be done in python also for the node?